### DISCUSSION TITLE
copy license sent activation link ok problems settings not found

### DISCUSSION THREAD

**Discussion by jolijojo** [aclap-dev/video-downloadhelper#1492] [Help and Q&A]
### Your question

Hello, I requested and received my license, then I activated the link received by email, but I can no longer find the usual settings or the backup folder. Does the license have a validity period or is it for life?
Thanks,
Jean-Pierre

### Addon & CoApp version and details

Hello, I requested and received my license, then I activated the link received by email, but I can no longer find the usual settings or the backup folder. Does the license have a validity period or is it for life?
Thanks,
Jean-Pierre

**Comment by GrampaWildWilly** [comment]
### I do not work for VDH.  I'm just another user.

![#01](https://github.com/user-attachments/assets/e565cff9-fb80-46c2-b601-5f0b5c3ebc3f)
![#02](https://github.com/user-attachments/assets/b2f784e6-b6a1-4a93-ad21-4fa496ff4780)
![#03](https://github.com/user-attachments/assets/15ebff76-e174-406f-87fd-f46f2bfcb800)

Once you do scroll those settings down, you will find the `Download directory`.  I assume that's what you intended to say when you said, "backup folder."  There is nothing called a "backup folder" in VDH.

The VDH license is for life & covers all the computers you personally own now & all the computers you will ever own.  But the license doesn't cover any computers you don't own, like those of family & friends.  Make them buy their own licenses.

**Reply by jolijojo** [reply]
Thank you for your help, I found the backup folder. The problem is that before I could enter the settings by right-clicking without problem. Now I have no options other than those displayed in my screenshots.
The photos you show me correspond well to my previous version known for this software.
Kind regards
Firefox 132.0.2
![page2](https://github.com/user-attachments/assets/c9005167-eb96-4d30-abf3-407afa8f6f44)
![page1](https://github.com/user-attachments/assets/338475d9-81d5-4a88-9972-e63f1f2d47ee)

**Comment by jolijojo** [comment]
Je rajoute une information, le nom du fichier vidéo sauvegardée n'est plus celui de la vidéo mais un numéro.

I'm adding some information, the name of the saved video file is no longer that of the video but a number.

**Reply by GrampaWildWilly** [reply]
I'm not sure I understand.  Do you not have a VDH icon on the toolbar of your browser window?

![#01](https://github.com/user-attachments/assets/1ff48794-c8ee-4cea-9a6e-5dff6d558521)

If that's not there, you need to . . .

![#02](https://github.com/user-attachments/assets/8588166c-317e-41e8-b569-108dc6288e1c)

. . . Click the `Extensions icon`.  It looks like a piece from a jigsaw puzzle.  This drops down a menu.  You should see an entry on this menu for VDH.  Click the `VDH extension settings` gear icon.  This opens yet another menu.  On that menu, click the `Pin to Toolbar` entry on this menu.  That should make the VDH icon appear on the browser toolbar.

I'm showing how this works in Firefox.  I don't know any other browser so you'll have to take what I'm saying here as ideas to try on your browser if you're not using Firefox.  It does look like you are using Firefox so that comment is more for other users who might be reading this.

Please stop using the term "backup folder."  There is no such thing in VDH.  If you mean `Download directory`, then please consistently say `Download directory`.  I am confused whenever you say "backup folder."  I don't know what you're talking about when you say "backup folder."  Please use the proper terms or explain what you mean.

When you click mouse button 2 to pop up a context menu on an object, sometimes there is no entry on the context menu for VDH & sometimes there is.  This seems to depend on a few factors.  First of all, it needs to be a video, not a link to a page where the video really is.  Even if you are on the page where the video really is, sometimes the embedded player on the web page has its own context menu & you don't get the standard browser context menu.  In that case, the player's own context menu won't show a VDH entry.  Even if you do get a VDH entry on the context menu, the VDH submenu that cascades out does not contain an entry to get into the VDH settings.  You have to use the VDH icon from the toolbar ... or from the VDH sidebar if you're using that.

As for your file naming issue, you'll have to provide at least one URL of a web page where there is a video that exhibits this behavior.  You should also post something like this: Windows 11 Pro 64-bit version 24H2 build 26100.2454, Firefox 132.0.2 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.  Then we will know what your environment is.  The various release numbers of the pieces of software are important.  The easy way to get this is from the VDH settings:

![#03](https://github.com/user-attachments/assets/cc551719-02c1-4dbf-842b-71d3029355cf)

Then paste that into a post here.  The operating system version number is not reported in that.  It just indicates whether you're running Windows or Mac or Linux without specifying a version number.  So you should supply that separately.

**Comment by jolijojo** [comment]
Finally it works thank you.
In fact I have always been used to the old configuration for which a right click was enough to have access to the parameters.
Now you have to move the icon to the toolbar to be able to access it. New simple manipulation that I did not know.
Have a good day or evening.
Jean-Pierre

Sorry for the bad computer vocabulary, not easy to use the translation when you speak French. Coming from the DOS era we actually spoke of directory, subdirectory main directory or root CD, CMD, REN...

**Reply by GrampaWildWilly** [reply]
Ah . . .  Le français.  Une langue assez difficile à maîtriser.  Je blague.  Moi, en anglophone que je sois, je ne connais pas les mots exacts qui paraissent dans le menu français de VDH.  Je m'y intéresse.  Pourrais-tu attacher une capture d'écran qui montre les mots français?  Peut-être ça prendrait 2 ou 3 captures parce qu'il s'agit d'un dialogue qu'il faut faire défiler.  Prends garde de cocher le petit x où se trouve ton clé de licence.  Çomme ça tu ne publieras pas ta licence en public.  Tu ne veux JAMAIS publier ta licence pour permettre à n'importe qui de la voler.  J'ai fait exactement cela dans un image ci-haut.

Tu peux déposer tes commentaires ici en n'importe quelle langue, y inclus bien sûr le français.  D'habitude GitHub offre un bouton pour traduire les commentaires.  Ne vois-tu pas ce bouton avec mes commentaires en l'autre langue?

**Reply by GrampaWildWilly** [reply]
Le bouton de traduction ne se trouve pas en haut de mon commentaire français.  GitHub n'est pas constant.  Je ne peux pas l'expliquer.  En tout cas, il y a toujours Google Traduction.
